Add option to update root record rather than a subdomain

chrislewisdev opened the pull request:
Heya! I recently used this tool when setting up a home webserver and it's been super handy, but I did encounter one limitation in that the tool assumes you want to update a subdomain rather than the root domain record. I added an option to support this.

Potential discussion points:
- I haven't updated the tests as they seem to require running against real Netlify data so I didn't feel comfortable running them at present
- If the root record is currently occupied by a NETLIFY record, updating will fail since we only check for A/AAAA records to delete prior to creation. Not sure if we want to account for those in the code or just leave it to the user to make sure there are no clashes first

Anyway, it is Christmas time so I don't expect any immediate attention on this. Happy holiday!

oscartbeaumont commented:
This looks good at first look. Would it be possible for you to fix the issues reported by CI (I think it's a linting thing)?

I am planning to do a heap of maintenance to this project in the next few weeks so I will merge it in when I do that.

Thanks for your contribution!

chrislewisdev commented:
There we go, I believe I've corrected the lint error though I still need to wait for you to approve the CI jobs to verify it 😅 While I was there I updated the env name for the new parameter to more closely match the actual name too.
